src/exampleWorkerManager.js

```diff
@@ -36,7 +36,18 @@ function resetWorker(grammar) {
   var examples = ohmEditor.examples.getExamples();
   Object.keys(examples).forEach(function(id) {
     var example = examples[id];
-    var match = grammar.match(example.text, example.startRule);
+    var match;
+    try {
+      match = grammar.match(example.text, example.startRule);
```

Comment:
Wouldn't it be possible to detect this condition before an error gets thrown? E.g.:

```
if (example.startRule in grammar.rules) {
  match = grammar.match(example.text, example.startRule);
} else {
  match = grammar.match(example.text);
}
```